x/foundation/sonar-project.properties

```diff
@@ -0,0 +1,15 @@
+sonar.projectKey=Finschia_finschia-sdk-x-foundation
+sonar.organization=finschia
+
+sonar.projectName=Finschia SDK - x/foundation
+sonar.project.monorepo.enabled=true
+
+sonar.sources=.
+sonar.exclusions=**/*_test.go
```

Comment:
`sonar.exclusions=**/*_test.go`
Can we move common exclusions above(root sonar-project.properties)?
And how about adding pb files `**/*.pb.go` to common as well?